### DISCUSSION TITLE
31-equal temperament tuning possible in GO?

### DISCUSSION THREAD

**Discussion by robinmcclellan** [GrandOrgue/grandorgue#1976] [Help]
Hello, I am new to GrandOrgue. I am hoping to retune to a 31-note equal-tempered scale, i.e. with the octave equally divided into 31 tones. I would like line up the notes in a row, so that one sounding octave takes up two-octaves-plus-a-fifth on the piano keys, like this:
![Screenshot 2015-09-09 08 57 20](https://github.com/user-attachments/assets/cd8eb581-38f5-4bb4-9acc-9e435d9c5039)

I am using an 88-key midi keyboard, so it is a single manual. 

Is this possible in GrandOrgue? If so, what do I need to do? I realize I may need to create the tuning in another software and import the temperament. If so, what file formats can be imported, and how to go about that? 

I have searched the forum and help pages. 

Many thanks!
Robin

**Comment by larspalo** [comment]
> Is this possible in GrandOrgue?

The automatic temperament distribution (re-tuning mechanism) of GrandOrgue is "hardwired" to 12 semitones per octave. So any automatic retuning like you suggests is not possible. It would be possible to feed GO with pre-tuned samples and use only the "Original temperament" but you'll have tons of warnings about the automatic temperament re-tuning as it certainly wouldn't match. Not to mention the work to put all the things together in a sample set.

>  If so, what file formats can be imported, and how to go about that?

Microsoft wave file format, or commonly said .wav files... Supported formats are 8 bit, 16 bit and 24 bit PCM or 32 bit IEEE float, either mono or stereo. The preferred sample rates are 44100 or 48000 Hz - GO supports any sample rate between 22000 and 96000 Hz.

If you're a programmer it would of course be possible to alter the temperament re-tuning system of GO, but I'd personally say it's not really "worth" it. I think you'd be much better off using a software synth (like for instance ZynAddSubFX) together with Scala as everything is more or less "ready to use" then.

**Comment by robinmcclellan** [comment]
Thank you very much for your help - this is exactly what I needed to know. I suppose the same is likely true in Hauptwerk as well... (knowing this is not a Hauptwerk forum)

Thanks again!
Robin
